shared/api/storage_pool.go

```diff
@@ -19,6 +19,10 @@ type StoragePool struct {
 	Name   string   `json:"name" yaml:"name"`
 	Driver string   `json:"driver" yaml:"driver"`
 	UsedBy []string `json:"used_by" yaml:"used_by"`
+
+	// API extension: clustering
+	State string   `json:"state" yaml:"state"`
+	Nodes []string `json:"nodes" yaml:"nodes"`
```

Comment:
Should be Location instead of Nodes

Reply:
Just to double check: "Location" singular?

Reply:
Went for "Location" for now.

Reply:
Locations would probably be best indeed given it's a list.

Reply:
done